### PR TITLE
Update documentation on pages to better introduce the Page Builder API

### LINKED ISSUE #857: Better drive users for Markdown/Html/Page Builder techniques to create pages.
**Description**

The doc should better drive user to one technology or another.

**Expected change**

Impact on the Page and the Visual Elements sections.

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Resolves #857.

+ Better title management

### CODE REVIEW

**Review by jrobinAV — APPROVED** [review]
View

File: docs/manuals/gui/binding.md [hunk 1]
```diff
@@ -68,22 +68,22 @@ where the page is defined, then the main module.
...
 These variables may not be exposed to other modules of the application.<br/>
 Global variables (the ones declared in the *\_\_main\_\_* module), on the other hand, can
 be used in all page content definitions. The module where the page is defined does not need
-to import the global variables if they are not used in the Python code of the module.
+to import the global variables if they are not used in the Python code for that module.
 
 !!! example "Example"
     Say you want to create a page that represents some evaluation of an expression using two
-    variables. The first variable would be global for the application. But the second variable
+    variables. The first variable would be global for the application. But, the second variable
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/binding.md:82]: don't like this ,

File: docs/manuals/gui/binding.md [hunk 2]
```diff
@@ -489,9 +489,9 @@ callback function.
...
 expression, you cannot update the *state* object directly. However, the `State^`
 class exposes the method `(State.)assign()^` that allows working around
-that limitation. Here is how you can change the value of the variable *toggle*
-directly from the Markdown definition of your page, within the body of the
-definition of a button:
+that limitation.<br/>
+Here is how you can change the value of the variable *toggle* directly from the Markdown definition
+of your page, within the body of the definition of a button:
 ```
 <|Toggle|button|on_action={lambda state: s.assign("toggle", not state.toggle)}|>
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/binding.md:496]: state: s.assign => state: state.assign

File: docs/manuals/gui/pages/builder.md [hunk 3]
```diff
@@ -126,12 +130,28 @@ with tgb.Page() as page:
...
 And the result is identical to what was shown above.
 
+!!! warning "Expression vs. value"
+    Note that you **can not** bind a variable by setting the property to the variable value, as in:
+    ```python
+       ...
+       tgb.input(first_name, label="First name")
+    ```
+    This code would indeed set the value of the default property of the `input` control to the value
+    of *first_name*. The control would correctly display the content of the string.<br/>
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/pages/builder.md:140]: First name or first_name ?

File: docs/manuals/gui/viselements/index.md [hunk 4]
```diff
@@ -205,7 +206,7 @@ fragment similar to:
     ```
 
     Prefixing a Boolean property name with "no&blank;", "not&blank;", "don't&blank;" or
-    "dont&blank;" and not setting the propery value is equivalent to setting that property to
+    "dont&blank;" and not setting the property value is equivalent to setting that property to
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/index.md:209]: good catch

**Pull request merged**